devtools/create_project/msvc.cpp

```diff
@@ -59,7 +59,7 @@ std::string MSVCProvider::getLibraryFromFeature(const char *feature, const Build
 		{      "sdl2", "SDL2.lib",                  "SDL2d.lib",     "winmm.lib imm32.lib version.lib setupapi.lib",    0 },
 		{      "libz", "zlib.lib",                  "zlibd.lib",     0,                                                 0 },
 		{       "mad", "mad.lib",                   0,               0,                                                 "libmad.lib" },
-		{   "fribidi", "libfribidi.lib",            0,               0,                                                 0 },
+		{   "fribidi", "fribidi.lib",               0,               0,                                                 0 },
```

Comment:
Won't this break the build for visual studio users using the precompiled libraries? (Which I believe is, practically all the visual studio users.)

Reply:
Which precompiled libraries? The `vcpkg` library is named `fribidi.lib`. Are you aware of any distribution that has it as `libfribidi.lib`?

Reply:
Yes, the one we provide: https://wiki.scummvm.org/index.php?title=Compiling_ScummVM/Visual_Studio

Reply:
Done